### DISCUSSION TITLE
Не работают плейлисты last.fm

### DISCUSSION THREAD

**Discussion by OxideManganese** [Chimildic/YaMuTools#94] [Помощь с ошибкой]
Сегодня заметил, что больше не создаются прей листы из last.fm (микс, рекомендовано и другие), каждый раз пишет "Треков не найдено"
Пробовал разные аккаунты ласта, так же разные браузеры, ошибка остаётся.

**Comment by visvital** [comment]
В моем случае, просто зависает на анимации с генерацией плейлиста, и ничего не происходит.

**Reply by OxideManganese** [reply]
Да, сейчас у меня так же

**Comment by Chimildic** [comment]
В Яндекс.Музыке изменилась страница с настройками. Теперь из нее нельзя достать логин lastfm, к которому привязан ваш профиль. Также они отключили страницу с импортом треков по названиям (раньше [тут](https://music.yandex.ru/import) была). Поэтому расширение спотыкается и плейлиста нет.

Возможно мне стоит убрать расширение из магазина. Потому что без поддержки многое сломано.

**Comment by Chimildic** [comment]
Просьба сходить на [форум яндекс музыки](https://yandexmusic.userecho.com/communities/6/topics/10680-vernite-import-muzyiki-v-tekstovom-formate) и проголосовать за возврат старого импорта (нужна регистрация)

**Comment by kisoid** [comment]
> Просьба сходить на [форум яндекс музыки](https://yandexmusic.userecho.com/communities/6/topics/10680-vernite-import-muzyiki-v-tekstovom-formate) и проголосовать за возврат старого импорта (нужна регистрация)

Проголосовал. Вот уроды, других слов нет... Только сегодня заметил... Хотел по-быстренькому замутить список перспективных мелодий за последние 90 дней из LastFm в текстовом файле, импортнуть и послушать... Хрен во всю морду!
